<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tri">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>711</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(255, 255, 255)

</string>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ajouter un produit</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajout</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>id_produit:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>reference</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>prix</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>date</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> nom_produit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>quantite</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_reference">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prix">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>160</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_date">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>250</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_quantite">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>290</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>310</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color: rgb(0, 0, 255)</string>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-5</y>
        <width>771</width>
        <height>461</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/272913746_990774311522570_960516843833593149_n (2).jpg</pixmap>
      </property>
     </widget>
     <zorder>label_10</zorder>
     <zorder>groupBox_2</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
     <zorder>le_id</zorder>
     <zorder>le_reference</zorder>
     <zorder>le_prix</zorder>
     <zorder>le_date</zorder>
     <zorder>le_nom</zorder>
     <zorder>le_quantite</zorder>
     <zorder>pb_ajouter</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>afficher un produit</string>
     </attribute>
     <widget class="QTableView" name="tabb">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>621</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>420</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>0</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>381</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="triparnom_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>380</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color:  rgb(0, 0, 255)</string>
      </property>
      <property name="text">
       <string>tri par nom croissant</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>supprimer un produit</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="title">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>id_produit:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_sup">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>200</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">   background-color: rgb(0, 0, 255)</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>300</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>701</width>
        <height>501</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/272913746_990774311522570_960516843833593149_n (2).jpg</pixmap>
      </property>
     </widget>
     <zorder>label_8</zorder>
     <zorder>groupBox</zorder>
     <zorder>label_7</zorder>
     <zorder>le_sup</zorder>
     <zorder>pb_supprimer</zorder>
     <zorder>progressBar</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>modifier un produit</string>
     </attribute>
     <widget class="QLineEdit" name="le_modifid">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_modifref">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prixmodif">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_modifdate">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>190</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_modifnom">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>230</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>id_produit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>reference</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>prix</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>date_ajout_produit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>nom_produit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_modifquantite">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>270</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>270</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>quantite</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_modifier">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>290</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color: rgb(0, 0, 255)</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_4">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>360</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>5</y>
        <width>721</width>
        <height>491</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/272913746_990774311522570_960516843833593149_n (2).jpg</pixmap>
      </property>
     </widget>
     <zorder>label_9</zorder>
     <zorder>le_modifid</zorder>
     <zorder>le_modifref</zorder>
     <zorder>le_prixmodif</zorder>
     <zorder>le_modifdate</zorder>
     <zorder>le_modifnom</zorder>
     <zorder>label_22</zorder>
     <zorder>label_23</zorder>
     <zorder>label_24</zorder>
     <zorder>label_25</zorder>
     <zorder>label_26</zorder>
     <zorder>le_modifquantite</zorder>
     <zorder>label_27</zorder>
     <zorder>pb_modifier</zorder>
     <zorder>progressBar_4</zorder>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>tri</string>
     </attribute>
     <widget class="QPushButton" name="triparnom">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color:  rgb(0, 0, 255)</string>
      </property>
      <property name="text">
       <string>tri par nom croissant</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>180</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color:  rgb(0, 0, 255)</string>
      </property>
      <property name="text">
       <string>tri par date croissante</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>240</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:  rgb(0, 0, 255)</string>
      </property>
      <property name="text">
       <string>tri par date decroissante</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_tri">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>51</y>
        <width>511</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <zorder>tableView_tri</zorder>
     <zorder>triparnom</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>pushButton_4</zorder>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>ravitaillement</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>maintenance</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>stat</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
